tensorflow/core/lib/core/threadpool.cc

```diff
@@ -24,6 +24,8 @@ limitations under the License.
 #include "tensorflow/core/platform/tracing.h"
 #include "tensorflow/core/platform/types.h"
 
+#include <fenv.h> // fesetround, FE_*
```

Comment:
`fenv.h` is not portable enough to go outside of `core/platform`.  You'll have to make something analogous to `ScopedFlushDenormal`.